demos/demo.cpp

```diff
@@ -0,0 +1,180 @@
+/**
```

Comment:
Was this file added intentionally?

Reply:
I needed to have a least one instance that get destroyed, for testing. I removed it.